apps/website/pages/components/progress-bar/code.tsx

```diff
@@ -0,0 +1,19 @@
+import Head from "next/head";
+import type { ReactElement } from "react";
+import ProgressBarPageLayout from "screens/components/progress-bar/ProgressBarPageLayout";
+import ProgressBarCodePage from "screens/components/progress-bar/overview/ProgressBarOverviewPage";
```

Comment:
The import is wrong: `import ProgressBarCodePage from "screens/components/progress-bar/code/ProgressBarCodePage ";`